doc/manpages/ocne-catalog.md

```diff
@@ -87,7 +87,7 @@ SUBCOMMANDS
 
 `mirror` [OPTIONS]...
   List the known container images used by applications in an application catalog and
-  optionally push them to a private registry. Images not present in the following Kubernetes 
+  optionally push them to a private registry or optionally download them to a .tgz file. Images not present in the following Kubernetes 
```

Comment:
start the "Images ..." sentence on a new line

Reply:
Ok!